### ISSUE TITLE
Two spaces and a new-line in Markdown doesn't translate into a new-line in draft-js

### ISSUE THREAD

**Issue by tilds** [Rosey/markdown-draft-js#87]
Hi, I was checking out the demo: http://www.roserobertson.me/markdown-draft-js/ and noticed that if I end a line in the markdown with two spaces and a new-line then the next line is just appended on to the first in the draft-js.
This is intended?
![image](https://user-images.githubusercontent.com/8914393/48718103-065f9680-ec1b-11e8-84f2-b67d52a7b831.png)

**Comment by Rosey** [comment]
Hmm yeah I think that is a bug, nice catch 👍 / 😢

**Comment by tilds** [comment]
I looked at it a bit more and the problem seems to be that 'hardbreak' is not handled in `markdown-to-draft.parseInline()`

Looking at the debug mode of the remarkable demo: https://jonschlinkert.github.io/remarkable/demo/ I get this:
```[
  {
    "type": "paragraph_open",
    "tight": false,
    "lines": [
      0,
      2
    ],
    "level": 0
  },
  {
    "type": "inline",
    "content": "First line  \nSecond line",
    "level": 1,
    "lines": [
      0,
      2
    ],
    "children": [
      {
        "type": "text",
        "content": "First line",
        "level": 0
      },
      {
        "type": "hardbreak",
        "level": 0
      },
      {
        "type": "text",
        "content": "Second line",
        "level": 0
      }
    ]
  },
  {
    "type": "paragraph_close",
    "tight": false,
    "level": 0
  }
]

**Comment by Rosey** [comment]
ahh, very nice. OK I'm a bit busy these days but I'll try to get to this, or if anyone else who is reading this issue and has the inclination they are welcome to take it on!

**Comment by Rosey** [comment]
Thanks to @kamilbielawski a fix for this has been released under `1.4.0`

**Comment by Rosey** [comment]
(and thanks @tilds for your detailed comment about what the issue was too 😄)
